scripts/connect-to-arc.sh

```diff
@@ -6,9 +6,8 @@ az account set --subscription=$SUBSCRIPTION > /dev/null 2>&1
 
 az extension remove --name connectedk8s
 
-az extension add --source https://shasbextensions.blob.core.windows.net/extensions/connectedk8s-$CONNECTEDK8S_VERSION-py2.py3-none-any.whl -y
-
-az -v 
+az extension add --source $AZ_EXTENSION_LOCATION/connectedk8s-$CONNECTEDK8S_VERSION-py2.py3-none-any.whl -y
```

Comment:
You should add the new env var to https://github.com/Azure/osm-azure/blob/main/.env.example